### DISCUSSION TITLE
The 1.9.0 release contains an API change.

### DISCUSSION THREAD

**Discussion by EdgarDegas** [guoyingtao/Mantis#334] [General]
I upgraded from 1.7.1 to 1.9.0 and the code won't compile. Then I realized there is an API change. I think we should avoid that in minor version upgrades.

If the APIs do need to be changed, you should increment the major version number (1.x -> 2.x).

**Comment by guoyingtao** [comment]
@EdgarDegas Thanks for pointing it out. Yes, you are correct. I did not follow the version rule very strictly before Mantis 2.X.X
